io.catenax.traction_battery_code/1.0.0/metadata.json

```diff
@@ -1 +1 @@
-{ "status" : "release"} 
+{ "status" : "STANDARDIZED"} 
```

Comment:
the value should be `standardize` and not `STANDARDIZED`

Reply:
@agg3fe In the governance overview the catgeory was written differently, if "standardize" is correct, can you please align it in the governance document aswell? otherwise it is a bit confusing.  https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md

Reply:
I updated my branch :)

Reply:
@jonbckr actually the correct status which should be mentioned in metadata.json is defined in the readme.md file. The one defined in Governance.md is the business value which reflects in the semantic hub. But yes, it's little confusing and will try to improve this on our side if possible.
For now I will keep these standardization related PRs on hold, because of some changes in the process and will merge once we are clear on these things.